### DISCUSSION TITLE
Is it possible to manage podman/container in cockpit-podman ?

### DISCUSSION THREAD

**Discussion by groovyman** [cockpit-project/cockpit#19928] [Q&A]
Dear friend,
thank you very much for allowing to create/delete container/pod in cockpit, this tool is really useful.
I was able to define a new pod containing a container inside, but left a typo in the definition of a volume
mount. Next i know i had to modify the settings, but i was unable to find an option to change the path.

So how could this be done ?

I found out, that cockpit saves the settings in a SQLLite db, but i would expect a dialog or a file to fix my settings, don't i?

**Comment by jelly** [comment]
Currently there is no edit option for containers see https://github.com/cockpit-project/cockpit-podman/issues/1293

You can however likely fix the typo with `podman --replace`
